static/vendor/node-snackbar/Gruntfile.js

```diff
@@ -0,0 +1,83 @@
+module.exports = function (grunt) {

```

Comment:
We don't need to use grunt.  We can just take the files we need and add them to the vendor directory.